database/test_schema.sql

```diff
@@ -160,6 +160,7 @@ CREATE TABLE `sessions` (
   `team_id` int(11) NOT NULL,
   `created_ts` timestamp NOT NULL DEFAULT 0,
   `last_access_ts` timestamp NOT NULL,
+   `last_page_access` text NOT NULL,
```

Comment:
No problem, we will remove this.  On a side note, we originally ran the format.sh script, but it changed a large number of upstream files which appear to be non-complaint.  Would you like a PR for re-formatting this code and existing code?